packages/story-editor/src/components/panels/shared/captionsPanelContent.js

```diff
@@ -76,11 +76,15 @@ function CaptionsPanelContent({
   clearFileText = __('Remove file', 'web-stories'),
 }) {
   const {
+    allowedMimeTypes: { caption: allowedTextMimeTypes },
     capabilities: { hasUploadMediaAction },
     MediaUpload,
   } = useConfig();
 
-  if (!hasUploadMediaAction && !tracks.length) {
+  if (
+    (!hasUploadMediaAction && !tracks.length) ||
```

Comment:
I didn't dig into the code that calls this etc... but any concern of hitting a `cannot read property length ...`  type of error here?

Reply:
Not an issue, I default the param to an empty array.

Reply:
Unlikely but a null value could still throw 

```
function append(value, array = []) {

  if(!arr.length){
      console.log("on no")
  }

  array.push(value)
  return array
}

append(1)  // [1]
append(2)  // [2], not [1, 2]

append(2, null) // VM121:3 Uncaught ReferenceError: arr is not defined
```

Reply:
Could happen with a null ya?

function append(value, myArr = []) {

  if(!myArr.length){
      console.log("on no")
  }

  myArr.push(value)
  return myArr
}

append(1)  // [1]
append(2)  // [2], not [1, 2]
append(2, null) // Cannot read properties of null (reading 'length')

Reply:
Theoretically yes, but all consumers using `<CaptionsPanelContent>` pass an empty array as a fallback, so this safeguard is not needed.